### PR TITLE
Added support for SHOW INDEX from table

### PR DESCRIPTION
**Pull request by jayant-kumar-yadav**
RDBMS like MySql and Mariadb allows checking indexes on a table using
SHOW INDEX from <table>

Added support for same.

### CODE REVIEW

File: src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt [hunk 1]
```diff
@@ -647,6 +647,9 @@ Statement SingleStatement() :
             LOOKAHEAD(ShowColumns())
             stm = ShowColumns()
             |
+            LOOKAHEAD(ShowIndex())
```
Comment (wumpz) [inline src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt:650]: Looking at this now, is the lookahead for each of these production branches really needed?

**Review by wumpz — CHANGES_REQUESTED** [review]
Thx for improving JSqlParser. Your PR is very promising. Could you address my two remarks?

**Review by manticore-projects — COMMENTED** [review]
No STAR imports please, its against policy.
Checkstyle would reject it anyways.

File: src/main/java/net/sf/jsqlparser/util/TablesNamesFinder.java [hunk 2]
```diff
@@ -13,15 +13,119 @@
 import java.util.List;
 import java.util.Map;
 
-import net.sf.jsqlparser.expression.*;
```
[the remark this answers is not in the document]
Comment (jayant-kumar-yadav) [reply src/main/java/net/sf/jsqlparser/util/TablesNamesFinder.java:16]: Done https://github.com/JSQLParser/JSqlParser/pull/1704/commits/75451ea00afea09b88bceb46ebcf4fdc0298dd45

File: src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt (hunk 1 above)
Comment (jayant-kumar-yadav) [reply src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt:650]: Just did for consistency

Comment (manticore-projects) [reply src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt:650]: It slows down the generated Parser and I would be surprised when a normal Lookahead like 2 or 3 was not sufficient (if we need a lookahead at all).

Lookaheads are very expensive!

File: src/main/java/net/sf/jsqlparser/parser/feature/Feature.java [hunk 3]
```diff
@@ -9,61 +9,8 @@
...
-import net.sf.jsqlparser.statement.select.TableFunction;
-import net.sf.jsqlparser.statement.select.Top;
-import net.sf.jsqlparser.statement.select.UnPivot;
-import net.sf.jsqlparser.statement.select.ValuesList;
-import net.sf.jsqlparser.statement.show.ShowTablesStatement;
-import net.sf.jsqlparser.statement.truncate.Truncate;
-import net.sf.jsqlparser.statement.update.Update;
-import net.sf.jsqlparser.statement.upsert.Upsert;
-import net.sf.jsqlparser.statement.values.ValuesStatement;
+import net.sf.jsqlparser.expression.*;
```
Comment (wumpz) [inline src/main/java/net/sf/jsqlparser/parser/feature/Feature.java:12]: again starimport

**Review by wumpz — CHANGES_REQUESTED** [review]
I approved you that your PRs do run a Github build action. There seem to be a JavaDoc problem introduced. Please correct your PR so that it is build successfully.

File: src/main/java/net/sf/jsqlparser/parser/feature/Feature.java (hunk 3 above)
Comment (jayant-kumar-yadav) [reply src/main/java/net/sf/jsqlparser/parser/feature/Feature.java]: Fixed

**Pull request merged**